TestProjects/AlembicImporter/Assets/Tests/Runtime/UnitTests/AlembicExportTest.cs

```diff
@@ -146,25 +151,34 @@ public class AlembicExportTest : AlembicTestBase {
         /// </summary>
         /// <param name="abcPath"></param>
         private void TestAbcImported (string abcPath) {
+#if UNITY_EDITOR
             AssetDatabase.Refresh ();
+#endif
 
             var absPath = GetAssetsAbsolutePath (abcPath);
 
             Assert.That (string.IsNullOrEmpty (absPath), Is.False);
             Assert.That (File.Exists (absPath));
 
+#if UNITY_EDITOR
```

Comment:
why are we skipping this test?

Reply:
the asset database is only available in the editor, can't import an alembic file in standalone

Reply:
please add comment to that effect.